systemd/mpathcount.service

```diff
@@ -8,4 +8,5 @@ StandardInput=socket
 StandardOutput=null
 StandardError=journal
 ExecStart=/usr/bin/sh -c '. /etc/xensource-inventory; while dd of=/dev/null bs=4096 count=1 status=none conv=noerror; do /opt/xensource/sm/mpathcount.py; done'
+ExecStartPost=/bin/bash -c 'if [ ! -f "/dev/shm/mpath_status" ]; then /usr/bin/python3 /opt/xensource/sm/mpathcount.py; fi'
```

Comment:
Maybe you can exec `/opt/xensource/sm/mpathcount.py` directly  when service restart no matter /dev/shm/mpath_status existing or not.

Reply:
It will slightly slow down the startup of the mpathcount service.

Reply:
What I concern is the mapth_status file won't update if the service stopped while udev notifies update event.
Leave to storage experts to help to review as well.

Reply:
You don't need this, certainly do not need to call it with `/usr/bin/python3` as the mpathcount.py has a `#!/usr/bin/python3` already

Reply:
Agree! I have deleted it.